### PR TITLE
Neuroimaging

### PR DESCRIPTION
**Pull request by saisha-r**
Neuroimaging/resources page

### CODE REVIEW

**Review by aaronjnewman — CHANGES_REQUESTED** [review]
This looks great, but a few changes - mostly stemming from when I decided it should be titled "neuroimaging", I forgot about the additions that were made which aren't neuroimaging per se:

- The section on lab meetings should be moved out of here. Probably to "join" but I'll review that one next :) 
- Assuming it's possible I'd like the navbar title to remain "neuroimaging", but the page title to be "Neuroimaging and Data Science". 
- If possible I'd like two headings at the top of the page, under the page title, that are links to the two main sections of the page: Neuroimgaing Tools, and Data Science (and rename the coding section to Data Science)
I also think I'll want to fiddle with the exact wording of the text more, but I think it will work better if you make these structural changes and then once we merge the branch I'll do my own to work on wording (and on my own schedule :)
- also not necessarily now (but maybe?) - this page should have images on it. At least, an image of an MRI (maybe from our Fiona photoshoot) and one of EEG (one of the pix of Katie with the EEG on all lit up, maybe one with Cindy in it too).

**Comment by saisha-r** [comment]
@aaronjnewman so, I was able to incorporate all your requests :D let me know what you think! I added images, and to make them side-by-side I had to use this html grid component. Unfortunately, with this, there doesn't seem to be a way for me to center the images when they are side-by-side. If it being left-aligned bothers you, let me know and maybe we can add a third image to each set to even it out? Or just make it one image for the EEG section and one image for MRI section, and center the images.

**Comment by aaronjnewman** [comment]
The images are great picks but I think they are too small as they are displayed. I think having them alongside paragraphs would work better, alternating image and text sides as on the Projects page. Look at the tages on the Projects page for guidance on doing this alignment.

I'm also realizing this website is waaay too wordy. This section is particulary bad - I think we want much simpler explanations of the techniques, aimed at potential participants from the general public, and kids. Later we could introduce maybe some "blog posts" that provide greater detail about the techniques as is currently on the neuroimaging page. 

I'm not asking you to rewrite it now (although you're welcome to :) but I'm flagging it while I think of it - and also to warn you that the number and size of paragraphs may change - so don't worry for now about aligning every paragraph with an image.

**Comment by saisha-r** [comment]
Alright, I have cut some text and formatted the pictures similarly to the projects page. Let me know what you think!

**Comment by aaronjnewman** [comment]
Looks great! However the pictures need to be resized/optimized as I'd mentioned for others on other pages. They take a long time to load and when I checked they're full 15 megapixel originals (5568 x 3712 pixels), whereas the size in the web page is only 816x552. In other words, the images are 33x larger than they need to be. So, please resize/ optimize compression (save as compressed jpeg, not PNG) for those images. Also make sure there are no spaces in the image filenames (calling out "DSC_3734 copy.JPG"), as that can cause problems.

I've updated the text so it's a bit simpler (I think I'm happy w it now), and added a couple of images for the data science section.

**Comment by saisha-r** [comment]
Loving the new font! 

I optimized and compressed the images I included. I am assuming the ones you added are already compressed? If yes, then I think we are good to go with this page! If not, I'd be happy to compress them

**Comment by aaronjnewman** [comment]
Yes, the ones I added are compressed and optimized, so good to go!
